djangoproject/static/js/djangoproject.js

```diff
@@ -64,3 +64,37 @@ document.querySelector('.menu-button').addEventListener('click', function () {
   this.classList.toggle('active');
   menu_el.classList.toggle('active');
 });
+
+// Update search input placeholder text based on the user's operating system
+(function () {
+  const el = document.getElementById('id_q');
+
+  if (!el) {
+    return;
+  }
+
+  const original_placeholder = el.getAttribute('placeholder');
+  const is_mac = navigator.userAgent.indexOf('Mac') !== -1;
+  const new_value = `${original_placeholder} (${is_mac ? '⌘' : 'Ctrl'} + K)`;
```

Comment:
Minor nit: on Apple platforms, keyboard shortcuts are usually formatted like ⌘ K without the plus sign (see pictures [here](https://developer.apple.com/design/human-interface-guidelines/keyboards#Platform-considerations) and [here](https://developer.apple.com/design/human-interface-guidelines/menus)).

```suggestion
  const new_value = `${original_placeholder} (${is_mac ? '⌘ K' : 'Ctrl + K'})`;
```

(I used narrow no-break space `\u202F` in there, which looks a little better than a full space and avoids any risk of weird wrapping.)

Reply:
Do you want to open a new issue about this since the PR was merged?

Reply:
Done, #1960